ext/standard/basic_functions.stub.php

```diff
@@ -79,6 +79,179 @@ function sort(array &$arg, int $sort_flags = SORT_REGULAR): bool {}
+function array_fill(int $start_key, int $num, $val) {}
+
+function array_fill_keys(array $keys, $val): array {}
+
+/**
+ * @param int|float|string $low
+ * @param int|float|string $high
+ * @param int|float $step
+ * @return array|false
+ */
+function range($low, $high, $step = 1) {}
+
+function shuffle(array &$arg): bool {}
+
+function array_pop(array &$stack) {}
+
+function array_shift(array &$stack) {}
+
+function array_unshift(array &$stack, ...$vars): int {}
+
+function array_splice(array &$arg, int $offset, int $length = UNKNOWN, $replacement = []): array {}
+
+function array_slice(array $arg, int $offset, ?int $length = null, bool $preserve_keys = false): array {}
+
+function array_merge(array ...$arrays): array {}
+
+function array_merge_recursive(array ...$arrays): array {}
+
+function array_replace(array $arr1, array ...$arrays): array {}
+
+function array_replace_recursive(array $arr1, array ...$arrays): array {}
+
+function array_keys(array $arg, $search_value = UNKNOWN, bool $strict = false): array {}
+
+/** @return int|string|null */
+function array_key_first(array $arg) {}
+
+/** @return int|string|null */
+function array_key_last(array $arg) {}
+
+function array_values(array $arg): array {}
+
+function array_count_values(array $arg): array {}
+
+/**
+ * @param int|string|null $column_key
+ * @param int|string $index_key
+ * @return array|false
+ */
+function array_column(array $arg, $column_key, $index_key = null) {}
+
+function array_reverse(array $input, bool $preserve_keys = false): array {}
+
+/** @return array|false */
+function array_pad(array $arg, int $pad_size, $pad_value) {}
+
+function array_flip(array $arg): array {}
+
+function array_change_key_case(array $input, int $case = CASE_LOWER): array {}
+
+function array_unique(array $arg, int $flags = SORT_STRING): array {}
+
+function array_intersect_key(array $arr1, array $arr2, array ...$arrays): array {}
+
+// tests break if $arr2 is variadic
+function array_intersect_ukey(array $arr1, array $arr2, $callback_key_compare_func): array {}
+
+function array_intersect(array $arr1, array ...$arrays): array {}
+
+// tests break if $arr2 is variadic
+function array_uintersect(array $arr1, array $arr2, $callback_data_compare_func): array {}
+
+function array_intersect_assoc(array $arr1, array ...$arrays): array {}
+
+// tests break if $arr2 is variadic
+function array_uintersect_assoc(array $arr1, array $arr2, $callback_data_compare_func): array {}
+
+function array_intersect_uassoc(array $arr1, array ...$arr2, callable $callback_key_compare_func): array {}
+
+function array_uintersect_uassoc(array $arr1, array ...$arr2, callable $callback_data_compare_func, callable $callback_key_compare_func): array {}
+
+function array_diff_key(array $arr1, array ...$arrays): array {}
+
+// tests break if $arr2 is variadic
+function array_diff_ukey(array $arr1, array $arr2, $callback_key_comp_func): array {}
+
+function array_diff(array $arr1, array ...$arrays): array {}
```

Comment:
I believe this one also requires at least two args, as well as the following functions.